synapse/tests/test_telepath.py

```diff
@@ -157,6 +160,10 @@ def test_telepath_basics(self):
 
             self.raises(s_exc.SynErr, prox.boom)
 
+        # Fini'ing a daemon fini's proxies connected to it.
+        self.true(evt.wait(2))
+        self.true(prox.isifini)
```

Comment:
`prox.isIfini` (extra i). How does this pass?

Reply:
oops. ends up in `__getattr__` for the proxy object.